src/main/resources/assets/ars_affinity/lang/en_us.json

```diff
@@ -9,5 +9,6 @@
   "ars_affinity.perk.PASSIVE_MOB_PACIFICATION": "%s ignore you",
   "ars_affinity.perk.PASSIVE_FIRE_THORNS": "%d%% chance to ignite enemies when attacked",
   "ars_affinity.perk.PASSIVE_HEALING_AMPLIFICATION": "Amplifies Heal by %d%% and reduces its food cost",
+  "ars_affinity.perk.PASSIVE_REVERBERATION": "While crouching, melee attacks send back knockback but cost %d%% of your mana",
```

Comment:
@cursoragent
"While crouching, melee attacks reflect knockback at the price of %d%% of your mana",